hello.ahk

```diff
@@ -15,103 +15,57 @@ Global VarL := ""
 DetectHiddenWindows, On ; BEcause most AHK script have their window hidden
 SetTitleMatchMode, 2
 
-GroupAdd, SuperMemo, ahk_class TBrowser ;Browser
-GroupAdd, SuperMemo, ahk_class TContents ;Content Window (Knowledge Tree)
-GroupAdd, SuperMemo, ahk_class TElWind ;Element window
-GroupAdd, SuperMemo, ahk_class TElDATAWind ;Element Data window
-GroupAdd, SuperMemo, ahk_class TSMMain ;Toolbar
-
-; these are the lines 1-4 on the menu
-    Menu, MyMenu, Add, L Window(L), MenuHandler  ;MenuHandler is a label -
-    Menu, MyMenu, Add, F SM18, MenuHandler
-    Menu, MyMenu, Add, G Discord, MenuHandler
-    Menu, MyMenu, Add, Run Chrome, MenuHandler
-
-    Menu, MyMenuOne, Add, Generic AI jailbreak, MenuHandlerOne
-    Menu, MyMenuOne, Add, Religious Debater, MenuHandlerOne
-    Menu, MyMenuOne, Add, Catholic Phd, MenuHandlerOne  ;MenuHandler is a label -
-    Menu, MyMenuOne, Add, Atheist Phd, MenuHandlerOne
-    Menu, MyMenuOne, Add, Takeaways from text, MenuHandlerOne
+function_initial_setup()
 
 ; RUN THESE PROGRAMS ON STARTUP
-  if (!WinExist("KeyLog (3).ahk")) {
-    Run, "..\Private_Folder\KeyLog (3).ahk"
-  }
-  if (!WinExist("NiftyWindows.ahk")) {
-    Run, "..\Private_Folder\NiftyWindows.ahk"
-  }
-  if (!WinExist("FastNavKeys.ahk")) {
-    Run, "..\Private_Folder\FastNavKeys.ahk"
-  }
-  if (!WinExist("3sec-tooltip-randomised.ahk")) {
-    Run, "..\Private_Folder\3sec-tooltip-randomised.ahk"
-  }
-  if (!WinExist("Anki.ahk")) {
-    Run, "..\Private_Folder\Anki.ahk"
-  }
-  if (!WinExist("something.ahk")) {
-    Run, "..\Private_Folder\something.ahk"
-  }
-  if (!WinExist("OCRv2.ahk")) {
-    Run, "..\Private_Folder\OCRv2.ahk"
-  }
-  if (!WinExist("ScreenBreak.ahk")) {
-    Run, "..\Private_Folder\ScreenBreak.ahk"
-  }
-  if (!WinExist("Socratic_Questions.ahk")) {
-    Run, "..\Private_Folder\Socratic_Questions.ahk"
-  }
-    if (!WinExist("Priorities.ahk")) {
-    Run, "..\Private_Folder\Priorities.ahk"
-  }
-  if (!WinExist("timer.ahk")) {
-    Run, "..\Private_Folder\timer.ahk"
-  }
-
-sendToVid(key, titlewin) {
```

Comment:
This is a function but still in the autoexecutable section. I recommend moving this below a return command to prevent unexpected function execution.